.github/workflows/ci.yml

```diff
@@ -26,8 +26,10 @@ jobs:
       with:
         fetch-depth: 0
 
+    # FIXME: Change to `uses: flox/install-flox-action@main' when that action
```

Comment:
@bryanhonof please patch this later today when `install-flox-action` is fixed. <3

Reply:
Fixed 😅